src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1896,9 +1889,8 @@ public FeedbackSessionResultsBundle getFeedbackSessionResultsForInstructorWithin
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, userEmail);
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, viewType);
        
-        return feedbackSessionsLogic.getFeedbackSessionResultsForInstructorWithinRangeFromView(feedbackSessionName,
-                                                                                               courseId, userEmail,
-                                                                                               range, viewType);
+        return feedbackSessionsLogic.getFeedbackSessionResultsForInstructorWithinRangeFromView(
+                feedbackSessionName, courseId, userEmail, range, viewType);
```

Comment:
Same as above:
Although 8 spaces works, maybe it's neater to indent 4 spaces from `[f]eedbackSessionsLogic...`?

Same for below...